.github/DISCUSSION_TEMPLATE/mend-renovate-app-support.yml

```diff
@@ -0,0 +1,61 @@
+name: Mend Renovate App Support
+description: Related specifically to uptime or performance of the Mend Renovate hosted app.
+body:
+  - type: checkboxes
+    id: hosted-app-checkbox
+    attributes:
+      label: I'm using the Mend Renovate hosted app.
+      description: Only use this discussion if you're using the Mend Renovate hosted app.
+      options:
+        - label: I'm using the Mend Renovate hosted app.
+          required: true
+
+  - type: dropdown
+    id: regression-error
+    attributes:
+      label: Was this something which used to work for you, and then stopped?
+      options:
+        - 'It used to work, and then stopped'
+        - 'I never saw this working'
+    validations:
+      required: true
+
+  - type: textarea
+    id: describe-problem
+    attributes:
+      label: Describe your problem.
+      description: 'Do not report any security concerns here. Email [renovate-disclosure@mend.io](mailto:renovate-disclosure@mend.io) instead.'
+    validations:
+      required: true
+
+  - type: textarea
+    id: debug-logs
+    attributes:
+      label: Relevant debug logs.
+      description: |
+        Log into [Renovate's app dashboard](https://app.renovatebot.com/dashboard) and locate the correct job log for when the problem occurred (e.g. when the PR was created).
+        Try to paste the *relevant* logs here, not the entire thing and not just a link to the dashboard (others don't have permissions to view them).
+        If you're not sure about the relevant parts of the log, then feel free to post the full log to a [Github Gist](https://gist.github.com/) and link to it.
+        Try to highlight the important logs into the issue itself.
+      value: |
+        <details><summary>Logs</summary>
+
+        ```
+        Copy/paste the relevant log(s) here, between the starting and ending backticks
+        ```
+
+        </details>
+    validations:
+      required: false
+
+  - type: dropdown
+    id: minimal-reproduction-repository
+    attributes:
+      label: Have you created a minimal reproduction repository?
+      description: Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.
+      options:
+        - 'No reproduction repository'
+        - 'No reproduction, but I have linked to a public repo where it occurs'
+        - 'I have linked to a minimal reproduction repository in the bug description'
+    validations:
+      required: true
```

Comment:
I think let's remove this part, as the quantity of people ticking yes is likely to be very low